app/models/metrics/algorithms.rb

```diff
@@ -107,5 +109,11 @@ def process_journals(event, matches)
       matches[:journal_exact] += 1 if journal_exact.count.positive?
       journal_exact
     end
+
```

Comment:
Please add docs. See https://github.com/MITLibraries/tacos/blob/main/app/models/metrics/algorithms.rb#L95-L110 for example syntax and check with `make docserver`